assets/src/edit-story/output/element.js

```diff
@@ -41,9 +47,9 @@ function OutputElement({ element }) {
       className="wrapper"
       style={{
         left: `${x}%`,
-        top: `${y}%`,
+        top: `${cta ? y - 100 * (1 - CTA_ZONE_PERCENT) : y}%`,
```

Comment:
Would it be better if we were to modify `box` calculation above? The box is passed around and I'm wondering if that could be an issue with it having the incorrect `height` in the sub-components. Also, the opposite could be true. So, please check what's better.

Reply:
I don't think so, the box calculation doesn't need to be skewed the wrapper is really an edge case here

Reply:
Sounds good.